### DISCUSSION TITLE
Cannot load such file

### DISCUSSION THREAD

**Discussion by tolaseadegbite** [ddnexus/pagy#528] [Q&A] [answered]
### Description

I got the following error after I added extras in pagy.rb in the initializers

Added and installed pagy
```
gem install pagy
```
application_controller.rb
```
include Pagy::Backend
```
application_helper.rb
```
include Pagy::Frontend
```
pagy.rb
```
require 'pagy/extras/support'
require 'pagy/extras/countless'
```

Error:
```
`require': cannot load such file -- pagy/extras/support (LoadError)
```

Server keeps stopping.

**Comment by benkoshy** [comment, marked as answer]
Try adding

`gem 'pagy', '~> 6.0'` to your Gemfile

Then run:

`bundle install`

Then see if you still get an error?

**Reply by tolaseadegbite** [reply]
This solved it
